### PR TITLE
release notes 2020.12

### LINKED ISSUE #4838: release notes 2020.12
relese note 2020.12

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #4838

### CODE REVIEW

**Review by redallen — APPROVED** [review]

**Review by evwilkin — APPROVED** [review]
LGTM

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -4,6 +4,65 @@ section: overview
 releaseNoteTOC: true
 ---
 
+## 2020.12 release notes (2020-09-17)
+- [@patternfly/react-catalog-view-extension@4.8.29](https://www.npmjs.com/package/@patternfly/react-catalog-view-extension/v/4.8.29)
+- [@patternfly/react-charts@6.9.7](https://www.npmjs.com/package/@patternfly/react-charts/v/6.9.7)
+- [@patternfly/react-core@4.50.4](https://www.npmjs.com/package/@patternfly/react-core/v/4.50.4)
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:10]: This version hasn't been released yet. `4.50.1` is the last `prerelease`.

**Review by redallen — CHANGES_REQUESTED** [review]

**Review by redallen — APPROVED** [review]

File: packages/react-docs/RELEASE-NOTES.md (hunk 1 above)
Comment (redallen) [reply packages/react-docs/RELEASE-NOTES.md]: I pushed a fix to this PR.

**Review by evwilkin — APPROVED** [review]
LGTM thanks!

**Pull request merged**